README.md

```diff
@@ -71,10 +71,10 @@ At this point, you should be ready to start your app in development mode.
 3. As long as you leave `SUPPRESS_SELF_INVITE=` blank in your `.env` you should be able to invite yourself from the homepage.
    - If you DO set that variable, then spoke will be invite-only and you'll need to generate an invite. Run:
```

Comment:
```suggestion
   - If you DO set that variable, then spoke will be invite-only and you'll need to generate an invite. Make sure that your invite hash is UUID encoded to pass the validation. Run:
```

Reply:
Why does it have to be UUID encoded?  Does it really not work without that?

Honestly, looking at the createOrganization and inviteByHash code in server/api/schema.js, I have no idea how this works at all -- e.g. createOrganization does a load by `inviteId` but that defeats the whole purpose of testing based on a hash, so this code needs some work.
If we have a volunteer interested in this, I can leave some notes -- we could fold fixing this into the warren improvement with a uuid per-campaign.

Reply:
Hi @schuyler1d - when i tried to setup an initial account (before the initial org was created), i found the initial signup failing, with a header saying `abc` was not a valid invite code. When i used a UUID in the SQL insert instead of `abc`, the initial invite code then did work.

I dug around the code and found that invite codes must be UUIDs, according to this validation code: https://github.com/MoveOnOrg/Spoke/blob/8642cd3413101c2df5e5c4be5f550ebe63d10ac7/src/server/local-auth-helpers.js#L49-L59

Might be worth adding some notes in ticket #1479.

Reply:
I am also seeing that when I experiment with this locally the UUID encoding works and is how the other invites in the DB are stored, while the non-encoded version does not work.

Reply:
ok, @kylehotchkiss -- thanks!  I missed where that was validated!
Let's definitely plan to fix this further, but I'm :+1: on the documentation change here until we do.